test/Transport.spec.ts

```diff
@@ -1,40 +1,40 @@
-import 'should'
-
 import { Transport } from './../src/Transport'
 import { TransportMessage } from './../src/Message'
 import { TestChannel } from './TestChannel'
-import * as sinon from 'sinon'
-import { SinonSpy } from 'sinon'
 
 const param = 'param'
 
 describe('Transport', () => {
 
-    context('subscriptions', () => {
+    describe('subscriptions', () => {
 
         const channel = new TestChannel()
         const stubMethods: Array<keyof TestChannel> = ['onConnect', 'onDisconnect', 'onData', 'onError']
-        stubMethods.forEach(method => sinon.stub(channel, method))
+        // tslint:disable-next-line
```

Comment:
oh... tslint... hello, old friend.

Reply:
Unsure how to fix this ~properly, good enough for now?

Reply:
Fine for now, but we should replace with eslint when we have the chance™